dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/program/hibernate/ProgramStageDataElement.hbm.xml

```diff
@@ -34,6 +34,8 @@
     <property name="renderType" type="jbValueRenderType" column="rendertype" />
 
     <property name="skipSynchronization" column="skipsynchronization" not-null="false" />
+    
+    <property name="skipAnalytics" column="skipanalytics" not-null="true" />
```

Comment:
If the column is non-null I wonder why we use `Boolean` and not `boolean` for the field.

Reply:
Yep, thought about it. Favored consistency here, as we use Boolean for all bool things in Program, ProgramStage, ProgramStageDataElement. Comes back to the question I raised previously on how we do default values, use of objects vs primitives, etc.